### ISSUE TITLE
mAP equals 0 after 10 epochs of training

### ISSUE THREAD

**Issue by RocketFlash** [hustvl/SparseInst#81]
Hi! Thank you for amazing work!

I am trying to train r50vd_giam_aug model on my dataset in COCO format. In categories I have only 1 class with id=1 and each segment has category_id=1. I changed NUM_CLASSES parameter to 1 and registered dataset in train_net.py script. Also I am training on 1 GPU with batch size = 8 with AMP (reduced LR to 0.000005). After 10 epochs of training mAP is still equals 0.  What am I doing wrong?

<img width="1267" alt="Screenshot 2022-09-16 at 16 10 36" src="https://user-images.githubusercontent.com/7099937/190616723-c32cf64f-c8e9-4634-9e2b-8189a14c1c05.png">

**Comment by RocketFlash** [comment]
Tried to train the model without AMP and mAP start growing. I've read other issues and it seems something wrong with AMP training

**Comment by wondervictor** [comment]
Hi @RocketFlash, thanks for your interest in SparseInst.
The learning rate is too small and you can use the default learning rate for an attempt. As for AMP, we suggest you use the  `soft` version of SparseInst to avoid the nan error.
